### PR TITLE
[shopsys] product does not open in new window

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Product detail was only page opening in new window. This behavior was weird and not well accepted by some customers so it has been removed.
|New feature| No 
|[BC breaks](https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/framework/src/Controller/Admin/ProductController.php [hunk 1]
```diff
@@ -165,13 +165,16 @@ public function editAction(Request $request, $id)
...
             $this->productFacade->edit($id, $form->getData());
 
-            $this->getFlashMessageSender()->addSuccessFlashTwig(
-                t('Product <strong>{{ product|productDisplayName }}</strong> modified'),
-                [
-                    'product' => $product,
-                ]
-            );
-            return $this->redirectToRoute('admin_product_edit', ['id' => $product->getId()]);
+            $this->getFlashMessageSender()
```
Comment (simara-svatopluk) [inline packages/framework/src/Controller/Admin/ProductController.php]: this behavior change is not explained in the US. I didn't find a criteria that would say "if the product is successfully edited, redirect to list". Am I right?

Comment (simara-svatopluk) [reply packages/framework/src/Controller/Admin/ProductController.php]: The same applies for a new product

File: packages/framework/src/Resources/scripts/admin/product.js [hunk 2]
```diff
@@ -19,11 +19,6 @@
         Shopsys.product.toggleIsUsingAlternateAvailability($outOfStockActionSelection.val() === Shopsys.constant('\\Shopsys\\FrameworkBundle\\Model\\Product\\Product::OUT_OF_STOCK_ACTION_SET_ALTERNATE_AVAILABILITY'));
 
         Shopsys.product.initializeSideNavigation();
-
-        $('#js-close-without-saving').on('click', function () {
```
Comment (simara-svatopluk) [inline packages/framework/src/Resources/scripts/admin/product.js]: I'm not sure whether it is a good idea to remove such functionality, the implementator can rely on it. If you need to remove it, please create upgrade instructions

Comment (TomasLudvik) [reply packages/framework/src/Resources/scripts/admin/product.js]: I have added upgrade note

File: packages/framework/src/Controller/Admin/ProductController.php (hunk 1 above)
Comment (TomasLudvik) [reply packages/framework/src/Controller/Admin/ProductController.php]: Well, reason for this was that product stuff in admin behaved different way than the others. I believe this is right and will discuss during BV, are you ok with that?

**Comment by simara-svatopluk** [comment]
Tests:
* the link for "closing without change" in product detail is now "back to overview", please revert the naming
* I tried to dump translations, but target `dump-translation` fails. Did you try to dump translations? Becuase there could be a new ones
* headline of product edit should be "Editing product: ", but you have "Editing product -"

File: packages/framework/src/Controller/Admin/ProductController.php (hunk 1 above)
Comment (simara-svatopluk) [reply packages/framework/src/Controller/Admin/ProductController.php]: sure

File: packages/framework/src/Resources/scripts/admin/product.js (hunk 2 above)
Comment (simara-svatopluk) [reply packages/framework/src/Resources/scripts/admin/product.js]: content is correct, but wrong upgrade.md, please move it to `upgrade-v8.0.0.md`

Comment (TomasLudvik) [reply packages/framework/src/Resources/scripts/admin/product.js]: Oh you are right!

**Comment by TomasLudvik** [comment]
> Tests:
> 
> * the link for "closing without change" in product detail is now "back to overview", please revert the naming
> * I tried to dump translations, but target `dump-translation` fails. Did you try to dump translations? Becuase there could be a new ones
> * headline of product edit should be "Editing product: ", but you have "Editing product -"

* same as in case of redirecting -> this should be unified as I did
* I have dumped translations (will create PR for fixing fail about dumping translations)
* again headline is same as for other entities (order, article ...)

**Comment by simara-svatopluk** [comment]
Please discuss the textation of "back to overview" and format of headline during BV, because there is no information about this in the acceptance criteria

File: packages/framework/src/Controller/Admin/ProductController.php (hunk 1 above)
Comment (TomasLudvik) [reply packages/framework/src/Controller/Admin/ProductController.php]: @PetrHeinz our current business validator agreed with me

**Pull request merged**